crates/hdk_derive/src/lib.rs

```diff
@@ -47,6 +47,9 @@ impl Parse for EntryDef {
                         match var.lit {
                             syn::Lit::Str(s) => required_validation_type = match s.value().as_str()
                             {
+                                "custom" => {
+                                    holochain_zome_types::validate::RequiredValidationType::Custom
+                                }
```

Comment:
There's a comment on line 64 saying "Options are: entry, sub_chain and full" which may need to be updated?